web-js/4.x/sketch-widget-event-explorer/README.md

```diff
@@ -0,0 +1,7 @@
+# Sketch widget event explorer
+
+This is a sample app to get familiar with the Sketch widget events lifecycle and the state changes:
+
+![sketch-widget-event-explorer-screenshot.png](assets/img/sketch-widget-event-explorer-screenshot.png)
+
+The code observes for most changes and will display some info on the right.
```

Comment:
Can we add a link to a live sample?  

Here is an example:
https://esri.github.io/developer-support/web-js/4.x/LayerList-with-attribute-table-toggle/